packages/devtools-plugin/src/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { createPlugin } from "@prodo/core/lib/plugins";
+import { Event } from "@prodo/core/lib/types";
+import DevTools, { DevMessage } from "@prodo/devtools-core";
+import { original } from "immer";
+import * as _ from "lodash";
+
+// tslint:disable-next-line:no-empty-interface
+export interface DevToolsConfig {}
+
+export interface DevToolsUniverse {
+  state: any;
+}
+
+const plugin = createPlugin<
+  DevToolsConfig,
+  DevToolsUniverse,
+  { state: any },
+  {}
+>("devtools");
+
+// Wrap user app in devtools, unless we're in test mode
+if (!process.env.JEST_WORKER_ID) {
```

Comment:
Can we use `process.env.NODE_ENV === "test"` instead. Users might use a different test runner than Jest